rpm/kubelet.spec

```diff
@@ -92,8 +92,8 @@ Release: %{RPM_RELEASE}
 Summary: Command-line utility for administering a Kubernetes cluster.
 Requires: kubelet >= 1.6.0
 Requires: kubectl >= 1.6.0
-Requires: kubernetes-cni >= 0.7.5
-Requires: cri-tools >= 1.11.0
+Requires: kubernetes-cni >= %{CNI_VERSION}
```

Comment:
because this spec is used across releases this needs to float to the lowest common denominator and explicitly be >= 0.6.0